src/lib/optimizer/strategy/or_to_union_rule.cpp

```diff
@@ -0,0 +1,34 @@
+#include <expression/expression_utils.hpp>
+
+#include "expression/logical_expression.hpp"
+#include "logical_query_plan/lqp_utils.hpp"
+#include "logical_query_plan/predicate_node.hpp"
+#include "logical_query_plan/union_node.hpp"
+#include "or_to_union_rule.hpp"
```

Comment:
I thought the linter wouldn't let you do this. The corresponding hpp should be at the very top.